composer.json

```
@@ -14,13 +14,13 @@
         "issues": "https://github.com/acquia/http-hmac-php/issues"
     },
     "require": {
-        "php": "^7.2",
+        "php": ">=7.3.0",
```

We have no idea if this will work with a theoretical PHP 9:

```suggestion
        "php": "^7.3 || ^8.0",
```